File: example_problems/hello/submissions/accepted/extra-file-c/test-hello.c
/*
 * This should give CORRECT on the default problem 'hello',
 * since the random extra file will not be passed to gcc.
 *
 * @EXPECTED_RESULTS@: CORRECT
 */

#include <stdio.h>

int main()
{
	char hello[20] = "Hello world!";
	printf("%s\n",hello);
	return 0;
}

File: example_problems/hello/submissions/accepted/extra-file-rb/test-hello.rb
# This should give CORRECT on the default problem 'hello',
# since the random extra file will not be passed.
#
# @EXPECTED_RESULTS@: CORRECT

if ENV['DOMJUDGE'] != '' then
  puts "Hello world!"
else
  puts "DOMJUDGE not defined"
  exit(1)
end
